OpenRA.Game/Graphics/SpriteRenderable.cs

```diff
@@ -59,7 +66,14 @@ float3 ScreenPosition(WorldRenderer wr)
 		public IFinalizedRenderable PrepareRender(WorldRenderer wr) { return this; }
 		public void Render(WorldRenderer wr)
 		{
-			Game.Renderer.WorldSpriteRenderer.DrawSprite(sprite, ScreenPosition(wr), palette, scale * sprite.Size);
+			var wsr = Game.Renderer.WorldSpriteRenderer;
+			if (palette.Palette.AllowTint)
```

Comment:
Thinking about this some more, this really should move off of the palette and into the sequences / sprite. We otherwise won't be able to disable tinting on RGBA sprites.